From 576dd11719baaff5e4e74bbd6662aa8604e70d7f Mon Sep 17 00:00:00 2001
From: HARAOUI Zakaria <z.hadraoui@gmail.com>
Date: Wed, 18 Feb 2026 05:32:42 +0100
Subject: [PATCH 3/3] =?UTF-8?q?Ignore=20venv=20pour=20=C3=A9viter=20de=20p?=
 =?UTF-8?q?ousser=20les=20secrets?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
 .gitignore | 33 ++++++++++++++++++++++++++++++++-
 1 file changed, 32 insertions(+), 1 deletion(-)

diff --git a/.gitignore b/.gitignore
index 7805df7..6382faf 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,2 +1,33 @@
 # .gitignore
-venv/venv/
+venv/
+__pycache__/
+*.pyc
+*.pyo
+*.pyd
+*.log
+*.sqlite3
+.DS_Store
+.idea/
+.vscode/
+*.env
+*.egg-info/
+dist/
+build/
+*.egg
+*.whl
+coverage/
+htmlcov/
+*.coverage
+*.coveragerc
+*.pytest_cache/
+*.mypy_cache/
+*.ipynb_checkpoints/
+*.vscode/
+*.sublime-project
+*.sublime-workspace
+*.bak
+*.tmp
+*.swp
+*.swo
+*.DS_Store
+*.egg-info/
\ No newline at end of file
-- 
2.43.0

